app/views/shortFormatPreSentenceReport/page1.scala.html

```diff
@@ -57,6 +57,7 @@ <h2 class="lede">Never lose your work again</h2>
         @inputHidden(reportForm("victimStatementInformationSource"))
         @inputHidden(reportForm("childrenServicesInformationSource"))
         @inputHidden(reportForm("policeInformationSource"))
+        @inputHidden(reportForm("sentencingGuidelinesInformationSource"))
```

Comment:
As mentioned previously we should think about pulling these repeated hidden fields out if possible